src/devices/tuya.ts

```diff
@@ -6101,6 +6101,85 @@ const definitions: Definition[] = [
             ],
         },
     },
+	{
+		fingerprint: [{
+			modelID: 'TS0601',
+			manufacturerName: '_TZE204_e9ajs4ft',
+		}],
+		model: 'CTL-R1-TY-Zigbee',
+		vendor: 'TuYa',
+		description: 'Creatrol - 24G radar human presence motion sensor.',
+		fromZigbee: [tuya.fz.datapoints],
+		toZigbee: [tuya.tz.datapoints],
+		exposes: [
+			e.illuminance_lux(),
+			e.presence(),
+			e.numeric('presence_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(100)
+			.withValueStep(1).withUnit('%').withDescription('presence_sensitivity'),
+	
+			e.numeric('detection_range', ea.STATE_SET).withValueMin(1.5).withValueMax(4.5).withValueStep(
+				0.1).withUnit('m').withDescription('Detection range'),
+	
+			e.numeric('detection_delay', ea.STATE_SET).withValueMin(1).withValueMax(600).withValueStep(1)
+			.withUnit('s').withDescription('presence detection delay'),
+	
+			e.numeric('illumin_max', ea.STATE_SET).withValueMin(0).withValueMax(2000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+	
+			e.numeric('illumin_min', ea.STATE_SET).withValueMin(0).withValueMax(1000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+			e.enum('presence_illumin_switch', ea.STATE_SET, ['disable', 'enable']).withDescription(
+				'Whether to enable illumination threshold detection'),
+			e.binary('light_switch', ea.STATE, "on", "off").withDescription(
+				'This state will determine the light on/off based on the lighting threshold and presence sensing'
+				),
+			e.enum('light_linkage', ea.STATE_SET, ['disable', 'enable']).withDescription('linkage_light'),
```

Comment:
Can you give some more context on what this option does?

Reply:
![58ea0949bee8eca494809d9c1e726d6](https://github.com/Koenkk/zigbee-herdsman-converters/assets/17340200/f4be7fc4-bf42-4ea8-a2ef-153baf95de07)
![bf2b4af5bb3904d15e0e4562875c285](https://github.com/Koenkk/zigbee-herdsman-converters/assets/17340200/2393101d-dd06-4bf2-bdbf-d775e5944852)